WowPacketParserModule.V8_0_1_27101/Parsers/UpdateHandler.cs

```diff
@@ -681,6 +695,13 @@ private static MovementInfo ReadMovementUpdateBlock(Packet packet, WowGuid guid,
                             key = UpdateFields.GetUpdateFieldName<PlayerField>(i);
```

Comment:
if (i > UpdateFields.GetUpdateField(PlayerField.PLAYER_END))
                                    goto case ObjectType.ActivePlayer;

Reply:
yes I did not add the named fields added to output in the first place, thats the reason why its different to a previous build of this branch. When comparing to any 7.x sniff it should be nearly identical. There's still a bug somewhere, but I did not look into it yet.

Reply:
blizzard just moved some part of UF from player to ActivePlayer, it's not a bug, u should just update code a bit more:
change this -
```C
                        case ObjectType.Player:
                        {
                            if (i < UpdateFields.GetUpdateField(UnitField.UNIT_END) || i < UpdateFields.GetUpdateField(UnitField.UNIT_FIELD_END))
                                goto case ObjectType.Unit;

                            key = UpdateFields.GetUpdateFieldName<PlayerField>(i);
                            break;
                        }
```
to this
```C
                        case ObjectType.Player:
                        {
                            if (i < UpdateFields.GetUpdateField(UnitField.UNIT_END) || i < UpdateFields.GetUpdateField(UnitField.UNIT_FIELD_END))
                                goto case ObjectType.Unit;
if (i > UpdateFields.GetUpdateField(PlayerField.PLAYER_END))
                                goto case ObjectType.ActivePlayer;

                            key = UpdateFields.GetUpdateFieldName<PlayerField>(i);
                            break;
                        }
```

Reply:
does not change anything. Output is still as invalid as before. Check PLAYER_DYNAMIC_FIELD_HEIRLOOMS/ACTIVEPLAYER_DYNAMIC_FIELD_HEIRLOOMS for reference.

Just noticed my updatefield dump was invalid, fixing it brb